scripts/stubsabot.py

```diff
@@ -320,12 +321,25 @@ async def suggest_typeshed_update(update: Update, session: aiohttp.ClientSession
             return
 
     body = "\n".join(f"{k}: {v}" for k, v in update.links.items())
-    body += """
 
-If stubtest fails for this PR:
-- Leave this PR open (as a reminder, and to prevent stubsabot from opening another PR)
-- Fix stubtest failures in another PR, then close this PR
-"""
+    stubtest_will_run = not meta.get("stubtest", {}).get("skip", False)
+    if stubtest_will_run:
+        body += textwrap.dedent(
+            """
+
+            If stubtest fails for this PR:
+            - Leave this PR open (as a reminder, and to prevent stubsabot from opening another PR)
+            - Fix stubtest failures in another PR, then close this PR
```

Comment:
```suggestion
            - Fix stubtest failures in another PR, then close this PR
            
            Note that you will need to close and re-open the PR in order to trigger CI
```
Might as well mention https://github.com/python/typeshed/issues/8434 while we're here